paddle/framework/tensor_util.cu

```diff
@@ -0,0 +1 @@
+./tensor_util.cc
```

Comment:
Why not move the code in tensor_util.cc to tensor_util.h. Or we can have a tensor_util_impl.h. It's a little strange to have a symbolic link.

Reply:
@QiJune Reyoung probably had an offline discussion with you ready. But looks like `.cu` is necessary to make nvcc pass the compilation...